services/api/routes/src/update.rs

```diff
@@ -1 +1,94 @@
+/*
+** Update route should handle updating the information of the existing user.
+** A user is allowed to update information such as follows:
+** Chosen notifier, set to null to disable notifications
+** The user's email address
+** User's telegram handle
+** User's enabled notifications
+**
+** Validate user by exists by using only the ID.
+** If the ID exists, then it can be validated.
+**
+** How can we ensure that the owner of the email | tg is making the update?
```

Comment:
Here are the details how I envisioned the authentication to look like:
1. User first registers an account and sets a notifier. During the registration process the user proves that they are the owner of the id they provided(the id will be a substrate address, and validation will be done by requesting the user to sign some 'welcome message'. The user will also prove that they are the notifier owner(e.g. when using email prove that they are the email owner).

2. After successfully proving, the server will give the user a JWT, which will be stored in the users browser.

3. When updating, the user will also send along the JWT, and after validating it we know that the user made the call and we can update the user data. However, when changing the notifier we need to validate that the user is actually the owner of the new selected notifier. This can be done the same way as in the registration process.

Part of the first step is getting implemented in this PR: #11